internal/providers/pke/pkeworkflow/update_cluster.go

```diff
@@ -185,6 +185,28 @@ func UpdateClusterWorkflow(ctx workflow.Context, input UpdateClusterWorkflowInpu
 		}
 	}
 
+	// Select images for new nodepools (if not specified)
+	{
+		activityInput := SelectImagesActivityInput{ClusterID: input.ClusterID, NodePools: input.NodePoolsToAdd}
+		err := workflow.ExecuteActivity(ctx, SelectImagesActivityName, activityInput).Get(ctx, &input.NodePoolsToAdd)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Select volume sizes
+	{
+		activityInput := SelectVolumeSizesActivityInput{
+			AWSActivityInput: awsActivityInput,
+			NodePools:        input.NodePoolsToAdd,
+		}
+
+		err := workflow.ExecuteActivity(ctx, SelectVolumeSizesActivityName, activityInput).Get(ctx, &input.NodePoolsToAdd)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Same comment as for the create flow.

Reply:
Fixed in e43d6fd3eedd4d530de6565a1a1150a30e56b9f1.